packages/atomic/src/pages/examples/ipx.html

```diff
@@ -38,7 +38,7 @@
             "
             section="search"
           >
-            <atomic-search-box></atomic-search-box>
+            <atomic-search-box style="padding-bottom: 0.875rem"></atomic-search-box>
```

Comment:
not super important since this is just an example, but I think we should generally avoid using `style=""` and instead use classes and style tags.

Reply:
Done ✅ !